neuralpp/test/quick_tests/symbolic/expression_test.py

```diff
@@ -317,3 +317,30 @@ def test_basic_z3_conversion():
     v2_times_neg_v = v2 * (-v)
     assert v2_times_neg_v.function.value == operator.mul
     assert v2_times_neg_v.arguments[1].function.value == operator.neg
+
+
+def test_multiply_bug():
+    """ From Winnie. """
+    from typing import Any
+    from neuralpp.symbolic.sympy_interpreter import SymPyInterpreter
```

Comment:
Shall we move the imports to the top of the file?

Reply:
I kept `from sympy.abc import a, b` there because I don't want global `a` and `b`.